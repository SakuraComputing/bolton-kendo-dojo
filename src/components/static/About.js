import React from 'react';
import { connect } from 'react-redux';
import Map, { Marker } from 'react-map-gl';
import Pin from '../../common/Pin';

const key = require('../../config/keys').apiMapKey;

const API_KEY = key;

const LATITUDE = 53.562614;
const LONGITUDE = -2.384833;

export class About extends React.Component {

  constructor(props) {
    super(props);
      this.state = {
        viewport: {
            latitude: LATITUDE,
            longitude: LONGITUDE,
            zoom: 14,
            bearing: 0,
            pitch: 0,
            width: '100%',
            height: 400,
        }
    };
  }

  render() {

    const { viewport } = this.state;

    const { club } = this.props.club;

    // const bollocks = club.address.map(alls => (
    //     <div>
    //         <div>{alls.clubName}</div>
    //     </div>
    // ));

    if(club === null)
        console.log('Props are', this.props.club.club[0].about);
    else 
        

    return (
        <div className="about-container">
            <div className="about-section" >
                <h2 className="about-header">Where we practice</h2>
                <div className="about-map">
                    <Map
                        {...viewport}
                        mapStyle="mapbox://styles/mapbox/streets-v10"
                        mapboxApiAccessToken={API_KEY}>
                        <Marker 
                            latitude={LATITUDE} 
                            longitude={LONGITUDE}
                        >
                            <Pin />       
                        </Marker>
                    </Map>
                    
                </div>
            </div>
            <div className="about-section" >
                <h2>Practice Times</h2>
            </div>
            <div className="about-section" >
                <h2>Contact</h2>
            </div>
        </div>
    )
  }
}

const mapStateToProps = (state) => ({
    club: state.club
});

export default connect(mapStateToProps)(About);
