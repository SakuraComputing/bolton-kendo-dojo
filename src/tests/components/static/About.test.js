import React from 'react';
import { shallow } from 'enzyme';
import { About } from '../../../components/static/About.js';

describe('About Bolton Kendo Dojo', () => {

    let about, club;

    beforeEach(() => {
        club = jest.fn();
        about = shallow(<About club={club}/>);
    })
    it('should render correctly', () => {
        expect(about).toMatchSnapshot();
    });
})
