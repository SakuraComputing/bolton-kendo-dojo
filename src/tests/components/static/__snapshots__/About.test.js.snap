// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Bolton Kendo Dojo should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About
    club={[MockFunction]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="about-section"
        >
          <h2
            className="about-header"
          >
            Where we practice
          </h2>
          <div
            className="about-map"
          >
            <InteractiveMap
              attributionControl={true}
              bearing={0}
              className=""
              clickRadius={0}
              container={<body />}
              disableTokenWarning={false}
              doubleClickZoom={true}
              dragPan={true}
              dragRotate={true}
              getCursor={[Function]}
              height={400}
              keyboard={true}
              latitude={53.562614}
              longitude={-2.384833}
              mapOptions={Object {}}
              mapStyle="mapbox://styles/mapbox/streets-v10"
              mapboxApiAccessToken="pk.eyJ1IjoiZXdhbmJkIiwiYSI6ImNqb2ozcnEycDAxMjkza3BhbjVpMWd0azUifQ.gRjILY0KoVrvFBY_Hvkv2w"
              maxPitch={60}
              maxZoom={24}
              minPitch={0}
              minZoom={0}
              onClick={null}
              onContextMenu={[Function]}
              onError={[Function]}
              onHover={null}
              onLoad={[Function]}
              onResize={[Function]}
              onStateChange={[Function]}
              onTransitionEnd={[Function]}
              onTransitionInterrupt={[Function]}
              onTransitionStart={[Function]}
              onViewStateChange={null}
              onViewportChange={null}
              pitch={0}
              preserveDrawingBuffer={false}
              preventStyleDiffing={false}
              reuseMaps={false}
              scrollZoom={true}
              style={null}
              touchAction="none"
              touchRotate={false}
              touchZoom={true}
              transitionDuration={0}
              transitionEasing={[Function]}
              transitionInterpolator={
                LinearInterpolator {
                  "around": undefined,
                  "propNames": Array [
                    "longitude",
                    "latitude",
                    "zoom",
                    "bearing",
                    "pitch",
                  ],
                }
              }
              transitionInterruption={1}
              visibilityConstraints={
                Object {
                  "maxPitch": 60,
                  "maxZoom": 24,
                  "minPitch": 0,
                  "minZoom": 0,
                }
              }
              visible={true}
              width="100%"
              zoom={14}
            >
              <Marker
                captureClick={true}
                captureDoubleClick={true}
                captureDrag={true}
                captureScroll={false}
                className=""
                draggable={false}
                latitude={53.562614}
                longitude={-2.384833}
                offsetLeft={0}
                offsetTop={0}
              >
                <Pin />
              </Marker>
            </InteractiveMap>
          </div>
        </div>,
        <div
          className="about-section"
        >
          <h2>
            Practice Times
          </h2>
        </div>,
        <div
          className="about-section"
        >
          <h2>
            Contact
          </h2>
        </div>,
      ],
      "className": "about-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h2
              className="about-header"
            >
              Where we practice
            </h2>,
            <div
              className="about-map"
            >
              <InteractiveMap
                attributionControl={true}
                bearing={0}
                className=""
                clickRadius={0}
                container={<body />}
                disableTokenWarning={false}
                doubleClickZoom={true}
                dragPan={true}
                dragRotate={true}
                getCursor={[Function]}
                height={400}
                keyboard={true}
                latitude={53.562614}
                longitude={-2.384833}
                mapOptions={Object {}}
                mapStyle="mapbox://styles/mapbox/streets-v10"
                mapboxApiAccessToken="pk.eyJ1IjoiZXdhbmJkIiwiYSI6ImNqb2ozcnEycDAxMjkza3BhbjVpMWd0azUifQ.gRjILY0KoVrvFBY_Hvkv2w"
                maxPitch={60}
                maxZoom={24}
                minPitch={0}
                minZoom={0}
                onClick={null}
                onContextMenu={[Function]}
                onError={[Function]}
                onHover={null}
                onLoad={[Function]}
                onResize={[Function]}
                onStateChange={[Function]}
                onTransitionEnd={[Function]}
                onTransitionInterrupt={[Function]}
                onTransitionStart={[Function]}
                onViewStateChange={null}
                onViewportChange={null}
                pitch={0}
                preserveDrawingBuffer={false}
                preventStyleDiffing={false}
                reuseMaps={false}
                scrollZoom={true}
                style={null}
                touchAction="none"
                touchRotate={false}
                touchZoom={true}
                transitionDuration={0}
                transitionEasing={[Function]}
                transitionInterpolator={
                  LinearInterpolator {
                    "around": undefined,
                    "propNames": Array [
                      "longitude",
                      "latitude",
                      "zoom",
                      "bearing",
                      "pitch",
                    ],
                  }
                }
                transitionInterruption={1}
                visibilityConstraints={
                  Object {
                    "maxPitch": 60,
                    "maxZoom": 24,
                    "minPitch": 0,
                    "minZoom": 0,
                  }
                }
                visible={true}
                width="100%"
                zoom={14}
              >
                <Marker
                  captureClick={true}
                  captureDoubleClick={true}
                  captureDrag={true}
                  captureScroll={false}
                  className=""
                  draggable={false}
                  latitude={53.562614}
                  longitude={-2.384833}
                  offsetLeft={0}
                  offsetTop={0}
                >
                  <Pin />
                </Marker>
              </InteractiveMap>
            </div>,
          ],
          "className": "about-section",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Where we practice",
              "className": "about-header",
            },
            "ref": null,
            "rendered": "Where we practice",
            "type": "h2",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <InteractiveMap
                attributionControl={true}
                bearing={0}
                className=""
                clickRadius={0}
                container={<body />}
                disableTokenWarning={false}
                doubleClickZoom={true}
                dragPan={true}
                dragRotate={true}
                getCursor={[Function]}
                height={400}
                keyboard={true}
                latitude={53.562614}
                longitude={-2.384833}
                mapOptions={Object {}}
                mapStyle="mapbox://styles/mapbox/streets-v10"
                mapboxApiAccessToken="pk.eyJ1IjoiZXdhbmJkIiwiYSI6ImNqb2ozcnEycDAxMjkza3BhbjVpMWd0azUifQ.gRjILY0KoVrvFBY_Hvkv2w"
                maxPitch={60}
                maxZoom={24}
                minPitch={0}
                minZoom={0}
                onClick={null}
                onContextMenu={[Function]}
                onError={[Function]}
                onHover={null}
                onLoad={[Function]}
                onResize={[Function]}
                onStateChange={[Function]}
                onTransitionEnd={[Function]}
                onTransitionInterrupt={[Function]}
                onTransitionStart={[Function]}
                onViewStateChange={null}
                onViewportChange={null}
                pitch={0}
                preserveDrawingBuffer={false}
                preventStyleDiffing={false}
                reuseMaps={false}
                scrollZoom={true}
                style={null}
                touchAction="none"
                touchRotate={false}
                touchZoom={true}
                transitionDuration={0}
                transitionEasing={[Function]}
                transitionInterpolator={
                  LinearInterpolator {
                    "around": undefined,
                    "propNames": Array [
                      "longitude",
                      "latitude",
                      "zoom",
                      "bearing",
                      "pitch",
                    ],
                  }
                }
                transitionInterruption={1}
                visibilityConstraints={
                  Object {
                    "maxPitch": 60,
                    "maxZoom": 24,
                    "minPitch": 0,
                    "minZoom": 0,
                  }
                }
                visible={true}
                width="100%"
                zoom={14}
              >
                <Marker
                  captureClick={true}
                  captureDoubleClick={true}
                  captureDrag={true}
                  captureScroll={false}
                  className=""
                  draggable={false}
                  latitude={53.562614}
                  longitude={-2.384833}
                  offsetLeft={0}
                  offsetTop={0}
                >
                  <Pin />
                </Marker>
              </InteractiveMap>,
              "className": "about-map",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "attributionControl": true,
                "bearing": 0,
                "children": <Marker
                  captureClick={true}
                  captureDoubleClick={true}
                  captureDrag={true}
                  captureScroll={false}
                  className=""
                  draggable={false}
                  latitude={53.562614}
                  longitude={-2.384833}
                  offsetLeft={0}
                  offsetTop={0}
                >
                  <Pin />
                </Marker>,
                "className": "",
                "clickRadius": 0,
                "container": <body />,
                "disableTokenWarning": false,
                "doubleClickZoom": true,
                "dragPan": true,
                "dragRotate": true,
                "getCursor": [Function],
                "height": 400,
                "keyboard": true,
                "latitude": 53.562614,
                "longitude": -2.384833,
                "mapOptions": Object {},
                "mapStyle": "mapbox://styles/mapbox/streets-v10",
                "mapboxApiAccessToken": "pk.eyJ1IjoiZXdhbmJkIiwiYSI6ImNqb2ozcnEycDAxMjkza3BhbjVpMWd0azUifQ.gRjILY0KoVrvFBY_Hvkv2w",
                "maxPitch": 60,
                "maxZoom": 24,
                "minPitch": 0,
                "minZoom": 0,
                "onClick": null,
                "onContextMenu": [Function],
                "onError": [Function],
                "onHover": null,
                "onLoad": [Function],
                "onResize": [Function],
                "onStateChange": [Function],
                "onTransitionEnd": [Function],
                "onTransitionInterrupt": [Function],
                "onTransitionStart": [Function],
                "onViewStateChange": null,
                "onViewportChange": null,
                "pitch": 0,
                "preserveDrawingBuffer": false,
                "preventStyleDiffing": false,
                "reuseMaps": false,
                "scrollZoom": true,
                "style": null,
                "touchAction": "none",
                "touchRotate": false,
                "touchZoom": true,
                "transitionDuration": 0,
                "transitionEasing": [Function],
                "transitionInterpolator": LinearInterpolator {
                  "around": undefined,
                  "propNames": Array [
                    "longitude",
                    "latitude",
                    "zoom",
                    "bearing",
                    "pitch",
                  ],
                },
                "transitionInterruption": 1,
                "visibilityConstraints": Object {
                  "maxPitch": 60,
                  "maxZoom": 24,
                  "minPitch": 0,
                  "minZoom": 0,
                },
                "visible": true,
                "width": "100%",
                "zoom": 14,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "captureClick": true,
                  "captureDoubleClick": true,
                  "captureDrag": true,
                  "captureScroll": false,
                  "children": <Pin />,
                  "className": "",
                  "draggable": false,
                  "latitude": 53.562614,
                  "longitude": -2.384833,
                  "offsetLeft": 0,
                  "offsetTop": 0,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                "type": [Function],
              },
              "type": [Function],
            },
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h2>
            Practice Times
          </h2>,
          "className": "about-section",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Practice Times",
          },
          "ref": null,
          "rendered": "Practice Times",
          "type": "h2",
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h2>
            Contact
          </h2>,
          "className": "about-section",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Contact",
          },
          "ref": null,
          "rendered": "Contact",
          "type": "h2",
        },
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="about-section"
          >
            <h2
              className="about-header"
            >
              Where we practice
            </h2>
            <div
              className="about-map"
            >
              <InteractiveMap
                attributionControl={true}
                bearing={0}
                className=""
                clickRadius={0}
                container={<body />}
                disableTokenWarning={false}
                doubleClickZoom={true}
                dragPan={true}
                dragRotate={true}
                getCursor={[Function]}
                height={400}
                keyboard={true}
                latitude={53.562614}
                longitude={-2.384833}
                mapOptions={Object {}}
                mapStyle="mapbox://styles/mapbox/streets-v10"
                mapboxApiAccessToken="pk.eyJ1IjoiZXdhbmJkIiwiYSI6ImNqb2ozcnEycDAxMjkza3BhbjVpMWd0azUifQ.gRjILY0KoVrvFBY_Hvkv2w"
                maxPitch={60}
                maxZoom={24}
                minPitch={0}
                minZoom={0}
                onClick={null}
                onContextMenu={[Function]}
                onError={[Function]}
                onHover={null}
                onLoad={[Function]}
                onResize={[Function]}
                onStateChange={[Function]}
                onTransitionEnd={[Function]}
                onTransitionInterrupt={[Function]}
                onTransitionStart={[Function]}
                onViewStateChange={null}
                onViewportChange={null}
                pitch={0}
                preserveDrawingBuffer={false}
                preventStyleDiffing={false}
                reuseMaps={false}
                scrollZoom={true}
                style={null}
                touchAction="none"
                touchRotate={false}
                touchZoom={true}
                transitionDuration={0}
                transitionEasing={[Function]}
                transitionInterpolator={
                  LinearInterpolator {
                    "around": undefined,
                    "propNames": Array [
                      "longitude",
                      "latitude",
                      "zoom",
                      "bearing",
                      "pitch",
                    ],
                  }
                }
                transitionInterruption={1}
                visibilityConstraints={
                  Object {
                    "maxPitch": 60,
                    "maxZoom": 24,
                    "minPitch": 0,
                    "minZoom": 0,
                  }
                }
                visible={true}
                width="100%"
                zoom={14}
              >
                <Marker
                  captureClick={true}
                  captureDoubleClick={true}
                  captureDrag={true}
                  captureScroll={false}
                  className=""
                  draggable={false}
                  latitude={53.562614}
                  longitude={-2.384833}
                  offsetLeft={0}
                  offsetTop={0}
                >
                  <Pin />
                </Marker>
              </InteractiveMap>
            </div>
          </div>,
          <div
            className="about-section"
          >
            <h2>
              Practice Times
            </h2>
          </div>,
          <div
            className="about-section"
          >
            <h2>
              Contact
            </h2>
          </div>,
        ],
        "className": "about-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h2
                className="about-header"
              >
                Where we practice
              </h2>,
              <div
                className="about-map"
              >
                <InteractiveMap
                  attributionControl={true}
                  bearing={0}
                  className=""
                  clickRadius={0}
                  container={<body />}
                  disableTokenWarning={false}
                  doubleClickZoom={true}
                  dragPan={true}
                  dragRotate={true}
                  getCursor={[Function]}
                  height={400}
                  keyboard={true}
                  latitude={53.562614}
                  longitude={-2.384833}
                  mapOptions={Object {}}
                  mapStyle="mapbox://styles/mapbox/streets-v10"
                  mapboxApiAccessToken="pk.eyJ1IjoiZXdhbmJkIiwiYSI6ImNqb2ozcnEycDAxMjkza3BhbjVpMWd0azUifQ.gRjILY0KoVrvFBY_Hvkv2w"
                  maxPitch={60}
                  maxZoom={24}
                  minPitch={0}
                  minZoom={0}
                  onClick={null}
                  onContextMenu={[Function]}
                  onError={[Function]}
                  onHover={null}
                  onLoad={[Function]}
                  onResize={[Function]}
                  onStateChange={[Function]}
                  onTransitionEnd={[Function]}
                  onTransitionInterrupt={[Function]}
                  onTransitionStart={[Function]}
                  onViewStateChange={null}
                  onViewportChange={null}
                  pitch={0}
                  preserveDrawingBuffer={false}
                  preventStyleDiffing={false}
                  reuseMaps={false}
                  scrollZoom={true}
                  style={null}
                  touchAction="none"
                  touchRotate={false}
                  touchZoom={true}
                  transitionDuration={0}
                  transitionEasing={[Function]}
                  transitionInterpolator={
                    LinearInterpolator {
                      "around": undefined,
                      "propNames": Array [
                        "longitude",
                        "latitude",
                        "zoom",
                        "bearing",
                        "pitch",
                      ],
                    }
                  }
                  transitionInterruption={1}
                  visibilityConstraints={
                    Object {
                      "maxPitch": 60,
                      "maxZoom": 24,
                      "minPitch": 0,
                      "minZoom": 0,
                    }
                  }
                  visible={true}
                  width="100%"
                  zoom={14}
                >
                  <Marker
                    captureClick={true}
                    captureDoubleClick={true}
                    captureDrag={true}
                    captureScroll={false}
                    className=""
                    draggable={false}
                    latitude={53.562614}
                    longitude={-2.384833}
                    offsetLeft={0}
                    offsetTop={0}
                  >
                    <Pin />
                  </Marker>
                </InteractiveMap>
              </div>,
            ],
            "className": "about-section",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Where we practice",
                "className": "about-header",
              },
              "ref": null,
              "rendered": "Where we practice",
              "type": "h2",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <InteractiveMap
                  attributionControl={true}
                  bearing={0}
                  className=""
                  clickRadius={0}
                  container={<body />}
                  disableTokenWarning={false}
                  doubleClickZoom={true}
                  dragPan={true}
                  dragRotate={true}
                  getCursor={[Function]}
                  height={400}
                  keyboard={true}
                  latitude={53.562614}
                  longitude={-2.384833}
                  mapOptions={Object {}}
                  mapStyle="mapbox://styles/mapbox/streets-v10"
                  mapboxApiAccessToken="pk.eyJ1IjoiZXdhbmJkIiwiYSI6ImNqb2ozcnEycDAxMjkza3BhbjVpMWd0azUifQ.gRjILY0KoVrvFBY_Hvkv2w"
                  maxPitch={60}
                  maxZoom={24}
                  minPitch={0}
                  minZoom={0}
                  onClick={null}
                  onContextMenu={[Function]}
                  onError={[Function]}
                  onHover={null}
                  onLoad={[Function]}
                  onResize={[Function]}
                  onStateChange={[Function]}
                  onTransitionEnd={[Function]}
                  onTransitionInterrupt={[Function]}
                  onTransitionStart={[Function]}
                  onViewStateChange={null}
                  onViewportChange={null}
                  pitch={0}
                  preserveDrawingBuffer={false}
                  preventStyleDiffing={false}
                  reuseMaps={false}
                  scrollZoom={true}
                  style={null}
                  touchAction="none"
                  touchRotate={false}
                  touchZoom={true}
                  transitionDuration={0}
                  transitionEasing={[Function]}
                  transitionInterpolator={
                    LinearInterpolator {
                      "around": undefined,
                      "propNames": Array [
                        "longitude",
                        "latitude",
                        "zoom",
                        "bearing",
                        "pitch",
                      ],
                    }
                  }
                  transitionInterruption={1}
                  visibilityConstraints={
                    Object {
                      "maxPitch": 60,
                      "maxZoom": 24,
                      "minPitch": 0,
                      "minZoom": 0,
                    }
                  }
                  visible={true}
                  width="100%"
                  zoom={14}
                >
                  <Marker
                    captureClick={true}
                    captureDoubleClick={true}
                    captureDrag={true}
                    captureScroll={false}
                    className=""
                    draggable={false}
                    latitude={53.562614}
                    longitude={-2.384833}
                    offsetLeft={0}
                    offsetTop={0}
                  >
                    <Pin />
                  </Marker>
                </InteractiveMap>,
                "className": "about-map",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "attributionControl": true,
                  "bearing": 0,
                  "children": <Marker
                    captureClick={true}
                    captureDoubleClick={true}
                    captureDrag={true}
                    captureScroll={false}
                    className=""
                    draggable={false}
                    latitude={53.562614}
                    longitude={-2.384833}
                    offsetLeft={0}
                    offsetTop={0}
                  >
                    <Pin />
                  </Marker>,
                  "className": "",
                  "clickRadius": 0,
                  "container": <body />,
                  "disableTokenWarning": false,
                  "doubleClickZoom": true,
                  "dragPan": true,
                  "dragRotate": true,
                  "getCursor": [Function],
                  "height": 400,
                  "keyboard": true,
                  "latitude": 53.562614,
                  "longitude": -2.384833,
                  "mapOptions": Object {},
                  "mapStyle": "mapbox://styles/mapbox/streets-v10",
                  "mapboxApiAccessToken": "pk.eyJ1IjoiZXdhbmJkIiwiYSI6ImNqb2ozcnEycDAxMjkza3BhbjVpMWd0azUifQ.gRjILY0KoVrvFBY_Hvkv2w",
                  "maxPitch": 60,
                  "maxZoom": 24,
                  "minPitch": 0,
                  "minZoom": 0,
                  "onClick": null,
                  "onContextMenu": [Function],
                  "onError": [Function],
                  "onHover": null,
                  "onLoad": [Function],
                  "onResize": [Function],
                  "onStateChange": [Function],
                  "onTransitionEnd": [Function],
                  "onTransitionInterrupt": [Function],
                  "onTransitionStart": [Function],
                  "onViewStateChange": null,
                  "onViewportChange": null,
                  "pitch": 0,
                  "preserveDrawingBuffer": false,
                  "preventStyleDiffing": false,
                  "reuseMaps": false,
                  "scrollZoom": true,
                  "style": null,
                  "touchAction": "none",
                  "touchRotate": false,
                  "touchZoom": true,
                  "transitionDuration": 0,
                  "transitionEasing": [Function],
                  "transitionInterpolator": LinearInterpolator {
                    "around": undefined,
                    "propNames": Array [
                      "longitude",
                      "latitude",
                      "zoom",
                      "bearing",
                      "pitch",
                    ],
                  },
                  "transitionInterruption": 1,
                  "visibilityConstraints": Object {
                    "maxPitch": 60,
                    "maxZoom": 24,
                    "minPitch": 0,
                    "minZoom": 0,
                  },
                  "visible": true,
                  "width": "100%",
                  "zoom": 14,
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "captureClick": true,
                    "captureDoubleClick": true,
                    "captureDrag": true,
                    "captureScroll": false,
                    "children": <Pin />,
                    "className": "",
                    "draggable": false,
                    "latitude": 53.562614,
                    "longitude": -2.384833,
                    "offsetLeft": 0,
                    "offsetTop": 0,
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "function",
                    "props": Object {},
                    "ref": null,
                    "rendered": null,
                    "type": [Function],
                  },
                  "type": [Function],
                },
                "type": [Function],
              },
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h2>
              Practice Times
            </h2>,
            "className": "about-section",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Practice Times",
            },
            "ref": null,
            "rendered": "Practice Times",
            "type": "h2",
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h2>
              Contact
            </h2>,
            "className": "about-section",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Contact",
            },
            "ref": null,
            "rendered": "Contact",
            "type": "h2",
          },
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
